File: packages/shared/src/enums/price-band.ts
export const PriceBand = {
  Free: "free",
  Budget: "budget",
  Mid: "mid",
  Premium: "premium",
} as const;

export type PriceBand = (typeof PriceBand)[keyof typeof PriceBand];

export const PRICE_BANDS = Object.values(PriceBand);

File: LetsGo/LetsGo/Models/Enums/PriceBand.swift
import Foundation

enum PriceBand: String, Codable, Hashable {
    case free
    case budget
    case moderate
    case premium

    var displayText: String {
        switch self {
        case .free: "Free"
        case .budget: "$"
        case .moderate: "$$"
        case .premium: "$$$"
        }
    }
}
